<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErosTitleBar</class>
 <widget class="QWidget" name="ErosTitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleBar
{
    background-color: #560d0d;
}

#lblTitle
{
	font-family: &quot;Noto Sans&quot;;
	color: #ffffff;
	font-size: 18px;
}

QToolButton, QToolButton::menu-indicator, QToolButton::up-arrow, QToolButton::down-arrow, QToolButton::left-arrow, QToolButton::right-arrow, QToolButton::pressed
{
    border: 0px solid transparent;
    background-color: transparent;
}

QPushButton, QPushButton::menu-indicator, QPushButton::up-arrow, QPushButton::down-arrow, QPushButton::left-arrow, QPushButton::right-arrow, QPushButton::pressed
{
    border: 0px solid transparent;
    background-color: transparent;
	color: #ffffff;
	background-color: #560d0d;
    min-width: 26px;
	padding: 0px;
}

QPushButton::hover
{
	background-color: #821313;
}

#btnClose {
	background-color: palette(window);
}

#btnClose::hover {
	background-color: #ffffff;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>23</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbMenu">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">Title</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>134</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="btnMin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/client/ui/min</normaloff>:/img/client/ui/min</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="btnMax">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/client/ui/max</normaloff>:/img/client/ui/max</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="btnClose">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>44</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/client/ui/close</normaloff>:/img/client/ui/close</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>20</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnMin</sender>
   <signal>clicked()</signal>
   <receiver>ErosTitleBar</receiver>
   <slot>minimizeBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMax</sender>
   <signal>clicked()</signal>
   <receiver>ErosTitleBar</receiver>
   <slot>maximizeBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ErosTitleBar</receiver>
   <slot>closeBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>maximizeBtnClicked()</slot>
  <slot>minimizeBtnClicked()</slot>
  <slot>closeBtnClicked()</slot>
 </slots>
</ui>
